akka-actor-typed/src/main/scala/akka/actor/typed/internal/StashBufferImpl.scala

```diff
@@ -128,6 +126,21 @@ import akka.util.OptionVal
 
   override def forEach(f: Procedure[T]): Unit = foreach(f.apply)
 
+  override def contains[U >: T](message: U): Boolean =
+    exists(_ == message)
+
+  override def exists(predicate: T => Boolean): Boolean = {
+    var hasElement = false
+    var node = _first
+    while (node != null && !hasElement) {
+      hasElement = predicate(node.message)
+      node = node.next
+    }
+    hasElement
+  }
+
+  override def anyMatch(predicate: Predicate[T]): Boolean = exists(predicate.test)
```

Comment:
maybe we should use `java.util.function.Predicate` here?

Reply:
hmm, I chose the Akka one because the `forEach` is using `akka.japi.function.Procedure`

On the other hand, I don't expect this kind of lambdas to throw exceptions so using the Java built-in `Predicate` should be ok.

Reply:
hmm, this was merged but we didn't change the function to use `java.util.function.Predicate`. 

Should I send another PR or not a big deal? 
As mentioned, `forEach` is already using `akka.japi` function.

cc @patriknw / @johanandren

Reply:
@octonato not a big deal, but change it to java.util.function.Predicate (since we haven't released it yet)